### ISSUE TITLE
Feature Suggestion - loadout manager

### ISSUE THREAD

**Issue by yozgarcia** [Numynum/TalentTreeTweaks#2]
Some classes have problems with the limited amount of builds, since the amount is shared between specs, it would be nice if your addon allowed us to store spec strings as a way to increase the amount of builds available per character. 

An example of the problem is the druid, I can only have 2 builds per spec, and between raid, m+, arena and rbg, and in the heal spec there are 4 different builds as a minimum

**Comment by SolidBrave** [comment]
Amen, main reason I am looking around for a addon regarding talent tree.

**Comment by raechu** [comment]
@Numynum would this be difficult to implement? My initial thought would be to save the talent loadouts yourself separately, and then when the user chooses a loadout saved by this addon, load it in as the 10th slot. Or the 1st, whatever really. Would just need to have a warning attached that everything in that slot gets overwritten by the addon.

I don't do wow addon dev though I'm just a normal boring dev so the API might not work that way of course. Just adding my 2 cents because I REALLY want extra slots and I came here hoping you'd done it with your addon.

**Comment by Numynum** [comment]
Complexity isn't the biggest issue tbh (though it's a bit more complicated than that)
I have a vague idea of what I want, and how to implement it
but sadly the big problem is time
I don't have a lot of time for addon development, and that's unlikely to change until mid/late January 🙁 
features that I'm adding now, are things I can cook up in an hour or two of work, spread over a week
my current guess for a good implementation of a loadout manager is closer to 10+ hours of work

So tl;dr, this is something I want to do, and it's been on my todo list for ages now, but the unfortunate reality, is that it'll stay there for now 🙁

**Comment by Numynum** [comment]
I finally found some time to work on this, and a beta version will be available on curse (once they approve the project, which takes anywhere between minutes and days for new projects)
https://www.curseforge.com/wow/addons/talent-loadout-manager
